go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/edgexfoundry/device-onvif-camera
 go 1.18
 
 require (
-	github.com/IOTechSystems/onvif v0.1.5
-	github.com/edgexfoundry/device-sdk-go/v2 v2.3.0
+	github.com/IOTechSystems/onvif v0.1.6
```

Comment:
Why is this being updated in the patch? i.e. are there bug fixes in this version that are really needed in the patch release?